.github/workflows/pull-request.yml

```
@@ -272,3 +271,53 @@ jobs:
+  Python-Compliance:
+    name: Check Python Code Style
+    runs-on: ubuntu-latest
+    needs: [Flutter-Codebase-Check]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)